components/admin_console/billing/billing_history.tsx

```diff
@@ -117,82 +116,108 @@ const getPaymentStatus = (status: string) => {
 };
 
 const BillingHistory: React.FC<Props> = () => {
-    const [billingHistory, setBillingHistory] = useState(billingInfo);
-
-    const showNoBillingHistoryView = () => {
-        setBillingHistory(undefined);
-    };
-
+    //const [billingHistory, setBillingHistory] = useState(billingInfo);
+    // TO-DO: Wire this up with real data
+    let billingHistory: undefined | Array<any>;
     const previousPage = () => {};
     const nextPage = () => {};
 
+    const paging = (
+        <div className='BillingHistory__paging'>
+            <FormattedMarkdownMessage
+                id='admin.billing.history.pageInfo'
+                defaultMessage='{startRecord} - {endRecord} of {totalRecords}'
+                values={{
+                    startRecord: 1,
+                    endRecord: 4,
+                    totalRecords: 4,
+                }}
+            />
+            <button
+                onClick={previousPage}
+                disabled={true}
+            >
+                <i className='icon icon-chevron-left'/>
+            </button>
+            <button
+                onClick={nextPage}
+                disabled={false}
+            >
+                <i className='icon icon-chevron-right'/>
+            </button>
+        </div>
+    );
+
     const billingHistoryTable = billingHistory && (
-        <table className='BillingHistory__table'>
-            <tr className='BillingHistory__table-header'>
-                <th>
-                    <FormattedMessage
-                        id='admin.billing.history.date'
-                        defaultMessage='Date'
-                    />
-                </th>
-                <th>
-                    <FormattedMessage
-                        id='admin.billing.history.description'
-                        defaultMessage='Description'
-                    />
-                </th>
-                <th className='BillingHistory__table-headerTotal'>
-                    <FormattedMessage
-                        id='admin.billing.history.total'
-                        defaultMessage='Total'
-                    />
-                </th>
-                <th>
-                    <FormattedMessage
-                        id='admin.billing.history.status'
-                        defaultMessage='Status'
-                    />
-                </th>
-                <th>{''}</th>
-            </tr>
-            {billingHistory.map((info: any) => (
-                <tr
-                    className='BillingHistory__table-row'
-                    key={info.id}
-                >
-                    <td>
-                        <FormattedDate
-                            value={info.date}
-                            month='2-digit'
-                            day='2-digit'
-                            year='numeric'
+        <>
```

Comment:
Moved paging into the history table so it would not show up on No History screen.